python/ray/tune/tune.py

```diff
@@ -197,3 +238,58 @@ def run_experiments(experiments,
             logger.error("Trials did not complete: %s", errored_trials)
 
     return runner.get_trials()
+
+
+def run_experiments(experiments,
+                    search_alg=None,
+                    scheduler=None,
+                    with_server=False,
+                    server_port=TuneServer.DEFAULT_PORT,
+                    verbose=2,
+                    resume=False,
+                    queue_trials=False,
+                    reuse_actors=False,
+                    trial_executor=None,
+                    raise_on_failed_trial=True):
+    """Runs and blocks until all trials finish.
+
+    Examples:
+        >>> experiment_spec = Experiment("experiment", my_func)
+        >>> run_experiments(experiments=experiment_spec)
+
+        >>> experiment_spec = {"experiment": {"run": my_func}}
+        >>> run_experiments(experiments=experiment_spec)
+
+        >>> run_experiments(
+        >>>     experiments=experiment_spec,
+        >>>     scheduler=MedianStoppingRule(...))
+
+        >>> run_experiments(
+        >>>     experiments=experiment_spec,
+        >>>     search_alg=SearchAlgorithm(),
+        >>>     scheduler=MedianStoppingRule(...))
+
+    Returns:
+        List of Trial objects, holding data for each executed trial.
+
+    """
+    # This is important to do this here
+    # because it schematize the experiments
+    # and it conducts the implicit registration.
+    experiments = convert_to_experiment_list(experiments)
+
+    trials = []
+    for exp in experiments:
+        trials += run(
```

Comment:
Just to make sure: is this expected to return flattened list of trials or an array of trials per experiment?

Reply:
I just tried to maintain the same behavior as before, which was to return a flattened list of trials